cmd/webhook/admission/admission_test.go

```diff
@@ -125,12 +125,12 @@ func TestRouteGroupAdmitter(t *testing.T) {
 		{
 			name:      "invalid routgroup multiple filters per json/yaml array item",
 			inputFile: "rg-with-multiple-filters.json",
-			message:   `single filter expected at \"status(201) -> inlineContent(\"hi\")\"\nsingle filter expected at \" \"`,
+			message:   `single filter expected at \"status(201) -> inlineContent(\\\"hi\\\")\"\nsingle filter expected at \" \"`,
 		},
 		{
 			name:      "invalid routgroup multiple predicates per json/yaml array item",
 			inputFile: "rg-with-multiple-predicates.json",
-			message:   `single predicate expected at \"Method(\"GET\") && Path(\"/\")\"\nsingle predicate expected at \" \"`,
+			message:   `single predicate expected at \"Method(\\\"GET\\\") && Path(\\\"/\\\")\"\nsingle predicate expected at \" \"`,
```

Comment:
Why do you need to quote this?

Reply:
Nothing specific, was just trying to make it more readable by quoting

Reply:
If you use backticks then you don't need to quote `"` anymore and %q will add `"` you strings.

Reply:
this is `%q` quotes

Reply:
true